lib/ipmi_fru.c

```diff
@@ -4804,10 +4803,15 @@ f_type, uint8_t f_index, char *f_string)
 		if (fru_area) {
 			free_n(&fru_area);
 		}
+		if (fru_data[fru_field_offset] == 0xc1) {
```

Comment:
Use `FRU_END_OF_FIELDS`.

Reply:
Why are you talking to yourself in your own PR ? You can just push an updated version.

Reply:
Just in case somebody reads the PR before I get to pushing a new version. We're actually now looking at some failures possibly resuting from editing certain fields/offsets/lengths we found in testing, so further changes are likely needed.

Reply:
I'm not surpised. As I said here: https://github.com/ipmitool/ipmitool/pull/191#issuecomment-659721389, the fru code is inherently awful. In general it's a "HERE BE DRAGONS" territory. I'm not sure I will accept your PR before the rewrite of fru is done.